.changeset/cyan-ears-wonder.md

```diff
@@ -0,0 +1,50 @@
+---
+'merchant-center-application-template-starter': minor
+'@commercetools-frontend/mc-scripts': minor
+'playground': minor
```

Comment:
Thanks that commercetools is your professional playground more than 10 years now @emmenko and we had even no "minor" issue with you 😄 

Hope for some - of course 10! - real 🍻  soon!